CHANGELOG.rst

```diff
@@ -3,9 +3,11 @@ CHANGELOG
 =========
 
 2.2.5dev
-=====
+========
```

Comment:
just don't forget to remove the extra === when it comes time to release :) (that's why I didn't bother adding it)

Reply:
Will do.